test/Riok.Mapperly.Tests/Helpers/MapperConfigurationBuilderTest.cs

```diff
@@ -36,6 +37,7 @@ public void ShouldMergeMapperConfigurations()
         mapper.UseReferenceHandling.Should().BeTrue();
         mapper.IgnoreObsoleteMembersStrategy.Should().Be(IgnoreObsoleteMembersStrategy.Source);
         mapper.RequiredMappingStrategy.Should().Be(RequiredMappingStrategy.Source);
+        mapper.PreferParameterlessConstructors.Should().BeTrue();
```

Comment:
This assertion is failing isn't it?
This test ensures that the configuration is merged correctly. To test this set different values in the `defaultMapperConfiguration` as in the `NewMapperConfiguration` and assert whether the resulting value is the one from the `NewMapperConfiguration`.

Reply:
okay I got that , so in this case `defaultMapperConfiguration  `has the value as `null `
and `NewMapperConfiguration  `has it as `true `, so the result is `true `